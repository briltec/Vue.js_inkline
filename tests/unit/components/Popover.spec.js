// import { mount } from '@vue/test-utils';
// import Popover from 'inkline/components/Popover';

describe('Components', () => {
    describe('Popover', () => {
        // let wrapper;

        beforeEach(() => {
            // wrapper = mount(Popover);
        });

        it('should render correctly', () => {
            expect(true);
        });
    });
});
