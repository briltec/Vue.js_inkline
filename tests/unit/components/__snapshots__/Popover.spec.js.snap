// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Popover should render correctly 1`] = `
<div class="popover-wrapper" id="popover">
  <div aria-haspopup="popover" aria-controls="popover"></div>
  <div class="popover" style="display: none;" name="zoom-in-top-transition" id="popover"><span class="arrow"></span>
    <!---->
    <!---->
    <!---->
  </div>
</div>
`;
